srcpkgs/sile/template

```diff
@@ -1,23 +1,29 @@
 # Template file for 'sile'
 pkgname=sile
-version=0.9.5.1
-revision=4
+version=0.10.9
+revision=1
 build_style=gnu-configure
+configure_args="--with-system-luarocks"
 hostmakedepends="pkg-config"
-makedepends="harfbuzz-devel lua51-devel lua51-lpeg lua51-luaexpat
- lua51-zlib lua51-luafilesystem lua51-luasocket lua51-luasec"
-depends="lua51-lpeg lua51-luaexpat lua51-zlib lua51-luafilesystem
- lua51-luasocket lua51-luasec"
+makedepends="fontconfig-devel harfbuzz-devel icu-devel lua54 lua54-cassowary
+	lua54-cosmo lua54-linenoise lua54-lpeg lua54-zlib lua54-cliargs
+	lua54-luaepnf lua54-luaexpat lua54-luafilesystem lua54-repl lua54-luasec
+	lua54-luasocket lua54-penlight lua54-stdlib lua54-vstruct"
+depends="fonts-sil-gentium libtexpdf lua54 lua54-cassowary lua54-cosmo
+	lua54-linenoise lua54-lpeg lua54-zlib lua54-cliargs lua54-luaepnf
+	lua54-luaexpat lua54-luafilesystem lua54-repl lua54-luasec lua54-luasocket
+	lua54-penlight lua54-stdlib lua54-vstruct"
```

Comment:
Put all the common dependencies in a `_luadeps` variable above.

Reply:
Can do. Unfortunately some of these Lua packages don't exist yet on Void, I think that's the main thing holding up this PR.